### PR TITLE
[Merged by Bors] - Split off usage guide into individual files and document MyID for multiple role groups

### LINKED ISSUE #557: Document myidOffset
### Problem

The cluster is unable to start when running multiple rolegroups without setting `myidOffset`, see https://github.com/stackabletech/zookeeper-operator/issues/546.

The property exists in the CRDs, but doesn't seem to be documented anywhere.

### Solution

Document that `myidOffset` must be set when running multiple rolegroups.

This should be in the Usage. We also want to use the opportunity to split the usage page into multiple pages, like it is done for trino: https://docs.stackable.tech/trino/stable/usage_guide/index.html

Then there'd be a new page "Running multiple role groups" where it would document that you have to set the myIdOffset

### Acceptance Criteria
- [ ] usage page split up
- [ ] new page "Running multiple role groups" created that documents setting the property

### PR DESCRIPTION
**Pull request by fhennig**
# Description

fixes #557 

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Review by maltesander — CHANGES_REQUESTED** [review]

File: docs/modules/usage_guide/pages/using_multiple_role_groups.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+= Using multiple role groups
+
+// abstract/summary
+When using multiple xref:home:concepts:roles-and-role-groups.adoc[role groups], every role group except one needs to be configured with an offset for the IDs of the ZooKeeper servers in that role group.
+
+ZooKeeper uses a unique ID called _myid_ to uniquely identify the servers in the cluster. The Stackable Operator uses the number of replicas in a role group to number the servers in the role group. If there is more than one role group, the other role groups need to be with an offset for the myid to prevent two servers from getting the same myid assigned.
+
+The offset is configured with the `myidOffset` config option on a role group:
```
[the remark this answers is not in the document]
Comment (fhennig) [reply docs/modules/usage_guide/pages/using_multiple_role_groups.adoc]: Thanks for the suggestion!

File: docs/modules/usage_guide/pages/configuration_environment_overrides.adoc [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        resources:
+          memory:
+            limit: '512Mi'
+          cpu:
+            max: '4'
+            min: '500m'
+          storage:
+            data:
+              capacity: '1Gi'
+----
```
[the remark this answers is not in the document]
Comment (fhennig) [reply docs/modules/usage_guide/pages/configuration_environment_overrides.adoc:110]: The storage bit above it as well I think. I'm moving it. But it was in the override section before, that's why it's in there. I didn't want to change the hierarchy of sections, just split them

**Review by maltesander — APPROVED** [review]
LGTM!

**Comment by fhennig** [comment]
bors merge

**Pull request closed without merging**